sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs

```diff
@@ -782,39 +848,76 @@ public partial class ScalingHostPoolReference
         public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
         public string TimeZone { get { throw null; } set { } }
     }
+    public partial class ScalingPlanPooledSchedulePatch : Azure.ResourceManager.Models.ResourceData
+    {
+        public ScalingPlanPooledSchedulePatch() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.DesktopVirtualization.Models.DayOfWeek> DaysOfWeek { get { throw null; } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.SessionHostLoadBalancingAlgorithm? OffPeakLoadBalancingAlgorithm { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.Time OffPeakStartTime { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.SessionHostLoadBalancingAlgorithm? PeakLoadBalancingAlgorithm { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.Time PeakStartTime { get { throw null; } set { } }
+        public int? RampDownCapacityThresholdPct { get { throw null; } set { } }
+        public bool? RampDownForceLogoffUsers { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.SessionHostLoadBalancingAlgorithm? RampDownLoadBalancingAlgorithm { get { throw null; } set { } }
+        public int? RampDownMinimumHostsPct { get { throw null; } set { } }
+        public string RampDownNotificationMessage { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.Time RampDownStartTime { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.DesktopVirtualizationStopHostsWhen? RampDownStopHostsWhen { get { throw null; } set { } }
+        public int? RampDownWaitTimeMinutes { get { throw null; } set { } }
+        public int? RampUpCapacityThresholdPct { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.SessionHostLoadBalancingAlgorithm? RampUpLoadBalancingAlgorithm { get { throw null; } set { } }
+        public int? RampUpMinimumHostsPct { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.Time RampUpStartTime { get { throw null; } set { } }
+    }
     public partial class ScalingSchedule
     {
         public ScalingSchedule() { }
         public System.Collections.Generic.IList<Azure.ResourceManager.DesktopVirtualization.Models.ScalingScheduleDaysOfWeekItem> DaysOfWeek { get { throw null; } }
         public string Name { get { throw null; } set { } }
         public Azure.ResourceManager.DesktopVirtualization.Models.SessionHostLoadBalancingAlgorithm? OffPeakLoadBalancingAlgorithm { get { throw null; } set { } }
-        public System.DateTimeOffset? OffPeakStartOn { get { throw null; } set { } }
```

Comment:
This is a breaking change that is not allowed, please use [custom code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/src/Customized/Models/ServiceAccountEncryptionProperties.cs) to add this property back and mark it as Obsolete and EditorBrowsableState.Never